shfmt/shfmt.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -eux
+
+fd -t x -e sh -e bash -x shfmt -ci -d -i 4
```

Comment:
Do we _really_ want to enforce a restriction on only checking executables? How about helper scripts `source`d from other executable scripts, this would exclude them.

Reply:
Fair enough, I do not think we have any helper scripts like that but the original check had `-t f` so I will keep it here. Thanks for the review!